bigflow/dagbuilder.py

```diff
@@ -75,15 +75,16 @@ def build_dag_operator(workflow_job, dependencies):
     is_delete_operator_pod=True,
     retries={retries},
     retry_delay=datetime.timedelta(seconds={retry_delay}),
-    dag=dag)
+    dag=dag,
+    execution_timeout={execution_timeout})
 """.format(job_var=job_var,
           task_id=task_id,
           docker_image = docker_repository+":"+build_ver,
           bf_job= workflow.workflow_id+"."+job.id,
           root_folder=root_package_name,
           retries=job.retry_count if hasattr(job, 'retry_count') else 3,
-          retry_delay=job.retry_pause_sec if hasattr(job, 'retry_pause_sec') else 60))
-
+          retry_delay=job.retry_pause_sec if hasattr(job, 'retry_pause_sec') else 60,
+          execution_timeout=timedelta(milliseconds=job.job_execution_timeout) if job.job_execution_timeout else None))
```

Comment:
We can use `datetime.timedelta` as type for `execution_timeout.
If it can't be `timedelta` for some reason (eg it comes from cmd-line / json etc), than it is better to use 'float seconds' (which is consistent with `time.time`, `time.sleep` and other functions).

Reply:
Straight int is simplier for user imo